src/dmd/dtoh.d

```diff
@@ -652,10 +652,11 @@ public:
 
                 case EnumKind.String, EnumKind.Enum:
                     buf.writestring("static ");
-                    writeEnumTypeName(type);
+                    auto target = determineEnumType(type);
+                    target.accept(this);
                     buf.printf(" const %s = ", vd.ident.toChars());
```

Comment:
`determineEnumType(memberType).accept(this);`

Reply:
`target` is used below and `memberType` is not defined here?

Reply:
my bad I missed that.